mcs/class/Mono.Debugger.Soft/Test/dtest.cs

```diff
@@ -4584,6 +4584,32 @@ Event step_out_await (string method, Event e)
 		assert_location(e, "ss_nested_arg");
 	}
 
+	[Test]
+	public void InspectFieldContent() {
+		//files.myBucket.GetEnumerator().get_Current().Key watching this generates an exception in Debugger
+		Event e = run_until("inspect_field_content");
```

Comment:
Could this test be simplified ?

Reply:
I assume only the enum field is needed, the Enumerator etc. stuff is not.

Reply:
No...This works completely fine. I will try another thing...
public class TestVargaz
    {
        private Position _currentPosition;
        private enum Position
        {
            BeforeFirst
        }
        internal TestVargaz()
        {
            _currentPosition = Position.BeforeFirst;
        }
    }

Reply:
It needs to be a struct i think.